lib/Listener/AlbumsManagementEventListener.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace OCA\Photos\Listener;
+
+use OCA\Photos\Album\AlbumMapper;
+use OCP\Files\File;
+use OCP\Files\Folder;
+use OCP\Files\Node;
+use OCP\EventDispatcher\Event;
+use OCP\EventDispatcher\IEventListener;
+use OCP\Files\Cache\CacheEntryRemovedEvent;
+use OCP\Share\Events\ShareDeletedEvent;
+use OCP\User\Events\UserDeletedEvent;
+
+class AlbumsManagementEventListener implements IEventListener {
+	private AlbumMapper $albumMapper;
+
+	public function __construct(AlbumMapper $albumMapper) {
+		$this->albumMapper = $albumMapper;
+	}
+
+	public function handle(Event $event): void {
+		if ($event instanceof CacheEntryRemovedEvent) {
```

Comment:
This is exactly the same as https://github.com/nextcloud/photos/blob/master/lib/Listener/NodeDeletedListener.php, so we should probably just listen to this event also there?

I had this in but you requested to remove it in #1598 IIRC